uperf-wrapper/Dockerfile

```diff
@@ -1,7 +1,13 @@
-FROM registry.access.redhat.com/ubi8:latest
+FROM registry.access.redhat.com/ubi8:latest as builder
+
+RUN dnf install -y --nodocs git autoconf automake gcc make
+RUN ln -s /usr/bin/python3 /usr/bin/python
+RUN git clone https://github.com/uperf/uperf /tmp/uperf
+RUN pushd /tmp/uperf && autoreconf --install &&  ./configure --disable-sctp ;make
```

Comment:
Might be interesting to include sctp due to the some of the telco asks....

Reply:
oh yes for some reason i wasn't able to get sctp to work, because of a few dependencies ignored it for now to make sure we can move forward, will look at adding dependencies later so we can enable with sctp, unless we'd like to wait for it first.. works either way to me..

Reply:
I managed to compile uperf with sctp by installing the RPM `lksctp-tools-devel`, available in the centos8 repository. The final container also needs the library `lksctp-tools` provided by the standard UBI8 repository.

Reply:
ack i'll make changes accordingly and see if it works/